File: Administrativo/spring-erp-servicio/net/royal/spring/programasocial/dominio/filtros/FiltroItem.cs
using net.royal.spring.framework.core.dominio;
using System;
using System.Collections.Generic;
using System.Text;

namespace net.royal.spring.rrhh.dominio.dto
{
    public class FiltroItem
    {
        public String codigo { get; set; }
        public String nombre { get; set; }
        public String estado { get; set; }
        public Int32? cantidad { get; set; }


        public ParametroPaginacionGenerico paginacion { get; set; }

        public FiltroItem()
        {
            paginacion = new ParametroPaginacionGenerico();
        }
    }
}

File: Administrativo/spring-erp-web/SpringNg/src/app/erp_module/programasocial/item/dominio/filtroItem.ts
import { ParametroPaginacionGenerico } from '../../../shared/dominio/ParametroPaginacionGenerico';

export class FiltroItem {

    constructor() {
        this.cantidad = 10;
        this.paginacion = new ParametroPaginacionGenerico();
        this.paginacion.cantidadRegistrosDevolver = this.cantidad;
    }
    paginacion: ParametroPaginacionGenerico;

    codigo: string;
    nombre: string;
    estado: string;

    cantidad: number;

}
